File: hamming.java
public class Solution {
    public int hammingWeight(int n) {
        int ans = 0;
        while (n != 0) {
            if ((n & 1) == 1) {
                ans++;
            }
            n = n >>> 1;
        }
        return ans;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int n = 11; // Replace 11 with your input value
        int result = solution.hammingWeight(n);
        System.out.println("Hamming weight: " + result);
    }
}

File: hamming.py
class Solution:
    def hammingWeight(self, n: int) -> int:
        ans = 0
        while n:
            if n & 1:
                ans += 1
            n >>= 1
        return ans

# Example usage:
solution = Solution()
n = 11  # Replace 11 with your input value
result = solution.hammingWeight(n)
print("Hamming weight:", result)
